app/main/session_parser.py

```diff
@@ -183,7 +184,7 @@ def restore_active_sessions():
             session.type = brew_session['type']
             session.session = brew_session['session']   # session guid
             session.id = -1                             # session id (interger)
-            # session.recovery = ''                     # find last step name
+            session.recovery = brew_session['recovery'] # find last step name
```

Comment:
extract the `recovery` data attribute from the lasts reported data log and adding it to the top level `session` dict for use in comparing the last known step name to the current step name.

Only used if the python server restarts mid session (aka active development, rpi power disruption, etc)